setup.py

```diff
@@ -32,7 +36,11 @@
     ],
     keywords='gwlf-e watershed hydrology',
     packages=find_packages(exclude=['tests']),
-    install_requires=[],
+    install_requires=[
+        'argparse == 1.2.1',
+        'numpy == 1.11.0',
+        'wsgiref == 0.1.2'
```

Comment:
I think it's just an artifact from creating a new virtualenv and doing `pip freeze`.